Improve getting started documentation (#37)
secureCodeBox/documentation: docs/getting-started/first-scans.md

## Installing the Nmap ScanType

In this guide, we'll use the [Nmap Port & Network Scanner](https://nmap.org), as it is fast and relatively easy to use. We can install the Nmap ScanType via Helm (Make sure to add the helm repo first. See [Installation](/docs/getting-started/installation)):

```bash
helm install nmap secureCodeBox/nmap
```

To verify or to see which ScanTypes are installed in your current Namespace you can run:

```bash
kubectl get scantypes
```

> This should print an output like this (maybe theversion differs):

Reviewer: ```suggestion
This should print an output like this (your version might differ):
```

```text
NAME   IMAGE
nmap   securecodebox/nmap:7.80
```

## Starting a Scan

> Now that we have the ScanType nmap installed, we are ready to start our first scan. A scanner like this Nmap ScanType is a namespaced CRD. That means you can install them in our own namespace and you're notrequired to have privileged access to the cluster. Also they are defined via YAML and so you can easily create your own ones.

Reviewer: ```suggestion
Now that we have the ScanType nmap installed, we are ready to start our first scan. A scanner, like this Nmap ScanType, is a namespaced CRD. That means you can install them in our own namespace and you're not required to have privileged access to the cluster. Also they are defined via YAML and so you can easily create your own ones.
```

    - scanme.nmap.org
```

To run this example save the YAML above to a local file named `nmap-scan.yaml` and then start the scan via kubectl:

```bash
kubectl apply -f nmap-scan.yaml
```

The scan is now starting up, you can track its progress using kubectl:

```bash
kubectl get scans
```

This should print an output like this:

```text
NAME                   TYPE   STATE      FINDINGS
nmap-scanme.nmap.org   nmap   Scanning
```

## Monitoring the Scan Execution

> When you apply a scan, the secureCodeBox Operator will create a [Kubernetes Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/) in your namespace. In this namespace scanner (in ourexample the Nmap scanner) will be executed inside a container. Once the scan has completed the container will terminate and no compute resources will be consumed anymore. You can view the status of this job by running:

Reviewer: ```suggestion
When you apply a scan, the secureCodeBox Operator will create a [Kubernetes Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/) in your namespace. In this namespace, the scanner (in our example the Nmap scanner) will be executed inside a container. Once the scan has completed the container will terminate and no compute resources will be consumed anymore. You can view the status of this job by running:
```

```bash
kubectl get jobs
```

This should give you an output like this

``` text
NAME                               COMPLETIONS   DURATION   AGE
parse-nmap-scanme.nmap.org-h8thd   1/1           30s        43m
scan-nmap-scanme.nmap.org-w66rp    1/1           10s        25s
```

:::note

> Your job names will be slightly different. Kubernetes generates a random suffix for each jobname to make them unique. In our case the suffix fir the scan job is `-w66rp` and forthe parse job is `-h8thd`.

Reviewer: ```suggestion
Your job names will be slightly different. Kubernetes generates a random suffix for each job name to make them unique. In our case the suffix fir the scan job is `-w66rp` and for the parse job is `-h8thd`.
```
